src/api/app/controllers/webui/requests/change_devel_projects_controller.rb

```diff
@@ -0,0 +1,40 @@
+module Webui
```

Comment:
Nitpick:

`devel_project_changes_controller.rb` would be more accurate since we're always requesting to one devel project at a time, not requesting to change multiple devel projects.

Reply:
Renamed.